app/addons/databases/actions.js

```diff
@@ -176,7 +176,7 @@ export default {
 
     databaseName = databaseName.trim();
 
-    const url = FauxtonAPI.urls('allDocs', 'app', app.utils.safeURLName(databaseName), '');
+    const url = FauxtonAPI.urls('allDocs', 'app', encodeURIComponent(databaseName), '');
```

Comment:
Same as aboce it seems its better to use `allDocsSantized` then you don't need to encodeURIComponent